pkg/providers/instancetype/types.go

```diff
@@ -41,12 +42,19 @@ import (
 )
 
 const (
-	MemoryAvailable = "memory.available"
-	NodeFSAvailable = "nodefs.available"
+	MemoryAvailable          = "memory.available"
+	NodeFSAvailable          = "nodefs.available"
+	PerformanceModeBurstable = "burstable"
```

Comment:
What do you think? Should this be "burstable" or "burst"?

Reply:
AWS calls those instances "burstable performance instances", so I kept it that way.
But it's up to you and other maintainers to decide.

Reply:
Got it. If this is the nomenclature that EC2 uses, I'd rather not stray from it